packages/addons/tailwindcss/index.ts

```diff
@@ -1,130 +1,50 @@
-import { defineAddon, defineAddonOptions } from '@sveltejs/cli-core';
+import { defineAddon } from '@sveltejs/cli-core';
 import { addImports } from '@sveltejs/cli-core/css';
-import { array, common, exports, imports, object } from '@sveltejs/cli-core/js';
-import { parseCss, parseScript, parseJson, parseSvelte } from '@sveltejs/cli-core/parsers';
+import { array, functions, imports, object, exports } from '@sveltejs/cli-core/js';
+import { parseCss, parseJson, parseScript, parseSvelte } from '@sveltejs/cli-core/parsers';
 import { addSlot } from '@sveltejs/cli-core/html';
 
-type Plugin = {
-	id: string;
-	package: string;
-	version: string;
-	identifier: string;
-};
-
-const plugins: Plugin[] = [
-	{
-		id: 'typography',
-		package: '@tailwindcss/typography',
-		version: '^0.5.16',
-		identifier: 'typography'
-	},
-	{
-		id: 'forms',
-		package: '@tailwindcss/forms',
-		version: '^0.5.10',
-		identifier: 'forms'
-	},
-	{
-		id: 'container-queries',
-		package: '@tailwindcss/container-queries',
-		version: '^0.1.1',
-		identifier: 'containerQueries'
-	}
-];
```

Comment:
amongst these 3 plugins, I believe `container-queries` is the only one that's now integrated in tailwind v4. we can keep the other two and add their plugin references to the stylesheet like so:
```css
@plugin "@tailwindcss/typography";
@plugin "@tailwindcss/forms";
```

Reply:
Hey @AdrianGonz97 I've been digging through the Tailwind v4 docs the last few days. There's no mention of plugins in the v4 docs as far as I can tell. They really aren't pushing them any more. My guess is more is (or will) be absorbed into core.

They confirmed container queries are now core:
https://tailwindcss.com/blog/tailwindcss-v4#container-queries
> We've brought container query support into core for v4.0, so you don't need the @tailwindcss/container-queries plugin anymore:

And at least from initial testing, form styles seemed improved by default, so those may have been added to the global reset. But not seeing anything official.

Typography I've not had a chance to test at all though.

Reply:
In the v4 doc I saw: https://tailwindcss.com/docs/functions-and-directives#plugin-directive
![image](https://github.com/user-attachments/assets/b5af9b89-a5f4-49f3-843c-aacc204e3d3d)

And it's maybe not the best to bring in `sv` an option to have "legacy" included? (even in option?)

But, open to change my mind ✌️
Let me know

Reply:
> This is likely the most official source at the moment:
> [tailwindlabs/tailwindcss#15715 (comment)](https://github.com/tailwindlabs/tailwindcss/discussions/15715#discussioncomment-11946588)

Thanks for linking that discussion. I added my two cents a few days ago. It looks like there is currently no way to include these plugins without using the "legcy" api mentioned above. Let's give them a few more days.

Currently though I would be leaning towards keeping the two plugins inside the add-on and linking them based on the docs provided above.

Reply:
If everything would work with the `legacy` mode why not. But it seems that it's not fully compatible.
Looking at https://github.com/tailwindlabs/tailwindcss-typography/issues/372 issue.

Reply:
Maybe I'm reading some of the comments wrongs in that issue, but apart from the missing documentation i'm unable to read from the comments that anyone using the `@plugin` api has problems with anything.

Reply:
You need to use back a `js` config to be able to tune it.
I mean, at this point, typography is not "native" v4. _(they are working on it)_

When I look at `sv`, I think: "Put me in the right track for x, y, z". 
I don't expect to start with a legacy way for doing things. If people needs `typography` and v4, they can do what on their own (it's userland).
When typography will be back to "native v4", yes I would be good to be able to add it directly from sv cli (like today with v3)

Reply:
See https://github.com/sveltejs/cli/pull/422#issuecomment-2642758906